docs/snippets/security-prereqs-binaries.md

```diff
@@ -0,0 +1,32 @@
+## Verifying Binaries
+
+Knative releases from 1.8 onwards are signed with [cosign](https://docs.sigstore.dev/cosign/overview).
+
+- **Verifying CLI Binaries**
+
+Knative releases from 1.8 onwards are signed with [cosign](https://docs.sigstore.dev/cosign/overview). You can use the following steps to verify our binaries.
+
+1. Download the files you want, and the `checksums.txt`, `checksum.txt.pem` and `checksums.txt.sig` files from the releases page:
+    ```sh
+    # this example verifies the 1.8.0 kn cli from the knative/client repository
+    wget https://github.com/knative/client/releases/download/knative-v1.8.0/checksums.txt
+    wget https://github.com/knative/client/releases/download/knative-v1.8.0/kn-darwin-amd64
+    wget https://github.com/knative/client/releases/download/knative-v1.8.0/checksums.txt.sig
+    wget https://github.com/knative/client/releases/download/knative-v1.8.0/checksums.txt.pem
+    ```
+1. Verify the signature:
+    ```sh
+    COSIGN_EXPERIMENTAL=1 cosign verify-blob \
+    --cert checksums.txt.pem \
+    --signature checksums.txt.sig \
+    checksums.txt
+    ```
+1. If the signature is valid, you can then verify the SHA256 sums match with the downloaded binary:
+    ```sh
+    sha256sum --ignore-missing -c checksums.txt
```

Comment:
```suggestion
1. Download the files you want, and the `checksums.txt`, `checksum.txt.pem`, and `checksums.txt.sig` files from the releases page, by running the commands:

    ```sh
    wget https://github.com/knative/client/releases/download/<kn-version>/checksums.txt
    wget https://github.com/knative/client/releases/download/<kn-version>/kn-darwin-amd64
    wget https://github.com/knative/client/releases/download/<kn-version>/checksums.txt.sig
    wget https://github.com/knative/client/releases/download/<kn-version>/checksums.txt.pem
    ```

    Where `<kn-version>` is the version of the CLI that you want to verify. For example, `knative-v1.8.0`.

1. Verify the signature by running the command:

    ```sh
    COSIGN_EXPERIMENTAL=1 cosign verify-blob \
    --cert checksums.txt.pem \
    --signature checksums.txt.sig \
    checksums.txt
    ```

1. If the signature is valid, you can then verify the `SHA256` sums match the downloaded binary, by running the command:

    ```sh
    sha256sum --ignore-missing -c checksums.txt
    ```

Reply:
add line spacing between steps for clarity

Reply:
steps should always have "by running the command" or similar, per https://github.com/knative/docs/blob/main/contribute-to-docs/style-guide/documenting-code.md#documenting-commands